web/app/helpers/maybe-query.ts

```diff
@@ -0,0 +1,28 @@
+import { helper } from "@ember/component/helper";
+
+export interface MaybeQuerySignature {
+  Args: {
+    Positional: [Record<string, unknown> | undefined];
+  };
+  Return: Record<string, unknown> | {};
+}
+
+/**
+ * Supplies an empty object if no is query provided.
+ * Avoids errors when passing empty `@query` values to LinkTos.
+ * Workaround for https://github.com/emberjs/ember.js/issues/19693
+ * Can be removed when we upgrade to Ember 3.28+
```

Comment:
Hmm looks like we're already on Ember 3.28? https://github.com/hashicorp-forge/hermes/blob/86f624b3c8212a59fc5e3af36e6680049f169206/web/package.json#L67

From that linked issue, I think we might have to upgrade to v4 for this to be fixed? (if that's correct, I'm suggesting that we just update the comment, and not that we perform the upgrade for this PR 🙂)